packages/vscode-extension/src/commands/SetupAccount.ts

```diff
@@ -233,37 +236,28 @@ export default class SetupAccount extends BaseAction {
 		});
 	}
 
-	private async getTokenId() {
+	private async getCertificateId() {
 		return window.showInputBox({
-			placeHolder: this.translationService.getMessage(MANAGE_ACCOUNTS.CREATE.SAVE_TOKEN.ENTER_TOKEN_ID),
+			placeHolder: this.translationService.getMessage(SETUP_ACCOUNT.CREATE.M2M.ENTER.CERTIFICATE_ID),
 			ignoreFocusOut: true,
-			password: true,
 			validateInput: (fieldValue) => {
 				let validationResult = InteractiveAnswersValidator.showValidationResults(
 					fieldValue,
-					InteractiveAnswersValidator.validateFieldIsNotEmpty
+					InteractiveAnswersValidator.validateFieldIsNotEmpty,
+					InteractiveAnswersValidator.validateFieldHasNoSpaces,
 				);
 				return typeof validationResult === 'string' ? validationResult : null;
 			},
 		});
 	}
 
-	private async getTokenSecret() {
-		return window.showInputBox({
-			placeHolder: this.translationService.getMessage(MANAGE_ACCOUNTS.CREATE.SAVE_TOKEN.ENTER_TOKEN_SECRET),
-			ignoreFocusOut: true,
-			password: true,
-			validateInput: (fieldValue) => {
-				let validationResult = InteractiveAnswersValidator.showValidationResults(
-					fieldValue,
-					InteractiveAnswersValidator.validateFieldIsNotEmpty
-				);
-				return typeof validationResult === 'string' ? validationResult : null;
-			},
+	private async getPrivateKeyFilePath() {
+		return window.showOpenDialog({
```

Comment:
Let's try to explicitly use some of the `window.showOpenDialog`  options so it will be clearer what can be returned.
I would like to make explicit use of:
- `OpenDialogOptions.canSelectMany=true`
- `OpenDialogOptions.canSelectFolders=false`
- `OpenDialogOptions.canSelectMany=false`

Reply:
I made the changes as you requested.

But just to note, I intentionally did not add those as they were optional and defaulted to the values we needed.